config.example.js

```diff
@@ -61,7 +61,6 @@ export const aptly = {
 export const database = 'mongodb://localhost/houston-dev'
 
 export const server = {
-  port: 3000,
```

Comment:
I wasn't aware you used it. It is set-able with a env variable, but I feel this setting is somewhat repetitive, so I'll make it optional.